File: MVPJava/app/src/functionalTests/java/com/vascome/fogtail/functional_tests/TestUtils.java
package com.vascome.fogtail.functional_tests;

import android.support.annotation.NonNull;
import android.support.test.InstrumentationRegistry;

import com.vascome.fogtail.FogtailApplication;

public class TestUtils {

    private TestUtils() {
        throw new IllegalStateException("No instances, please");
    }

    @NonNull
    public static FogtailApplication app() {
        return (FogtailApplication) InstrumentationRegistry.getTargetContext().getApplicationContext();
    }
}

File: MosbyMVPKotlin/app/src/functionalTests/java/com/vascome/fogtail/functionalTests/TestUtils.kt
package com.vascome.fogtail.functionalTests

import android.support.test.InstrumentationRegistry

import com.vascome.fogtail.FogtailApplication

class TestUtils private constructor() {

    init {
        throw IllegalStateException("No instances, please")
    }

    companion object {

        fun app(): FogtailApplication {
            return InstrumentationRegistry.getTargetContext().applicationContext as FogtailApplication
        }
    }
}
